### ISSUE TITLE
Send a clip to OS (Windows) clipboard

### ISSUE THREAD

**Issue by Chealer** [sabrogden/Ditto#574]
Some applications can read the OS clipboard for a purpose other than pasting. For example, the Eclipse IDE's _Open from clipboard_ feature (`Ctrl` + `Shift` + `V`) opens a symbol matching the clipboard's content.

Although Eclipse could arguably offer a dialog to search with a regular text input, Ditto could facilitate using such features by offering a feature to move a clip to the Microsoft Windows clipboard. That way, even if a clip's content is not in the clipboard, it would be efficient to use it.
With Eclipse, one could:
- Bring up Ditto
- Press `Apps/Menu`
- Send To → Clipboard
- Press `Ctrl` + `Shift` + `V`

Ditto unfortunately does not allow this as of version 3.24.246.0.

**Comment by PzaThief** [comment]
Ditto send clip to OS clipboard when you copy clip in quick list.

In general OS clipboard clip and Ditto clip(latest) is equal (unless you don't disconnect clipboard with Ditto or something else)
In this case you might not need to send Ditto's clip.

So, I think your common issue case seems about older clips in Ditto.
If it is, I suggest to use copy clip in quick list via shortcut(simply Ctrl + V).
This is because "right click and select send to" action is more heavy (in user experiance, not technically) than copy action.

It's a unnecessary addition, but the Send To function is actually a transfer function over a network, so it's a little out of context with what you said.

**Comment by Chealer** [comment]
Hi @PzaThief 

Indeed, this issue is about "older clips".

Can you clarify what you mean by "quick list"?

**Comment by PzaThief** [comment]
Ah, I mean Quick paste list that popped up when you use Activate Ditto shortcut.
Just select clip via up or down arrow key or left click and then copy it.
then Ditto will send the clip to OS.

**Comment by Chealer** [comment]
Thanks @PzaThief , but that does not work, with either 3.24.214.0 or 3.24.246.0 (the current version).

**Comment by PzaThief** [comment]
Could you let me know how did you try?
I tried with the same version and Eclipse ide, it works.
the popup must disappear if you copied(Ctrl+C) clip.

**Comment by Chealer** [comment]
> Could you let me know how did you try?

Of course:

1. I focus an editable field, such as this one.
2. I press Ditto's `Ctrl`+backtick shortcut.
3. I click clip #2
4. I press `Esc` to close Ditto.
5. I press Windows's `Ctrl`+`V` shortcut to paste the OS clipboard.

This results in the contents of clip #1 being pasted, rather than #2's.

> I tried with the same version and Eclipse ide, it works.

Thanks. I did not try with Eclipse IDE (I used Firefox on GitHub).

**Comment by PzaThief** [comment]
Oh. I make a communication error. lol
don't close Ditto manually(remove 4th step) and copy again the clip item.
Select clip item → copy it → paste it where you want.
Ditto will be disappear by itself when you copy clip item.

**Comment by Chealer** [comment]
Ah, well thanks anyway @PzaThief but I do not want to paste the clip. This issue is about sending a clip to the OS clipboard.

**Comment by PzaThief** [comment]
paste is optional. if you copy clip item in list, the clip will be sent to OS clipboard.

**Comment by Chealer** [comment]
I am sorry but that does not work. If I click an item, nothing happens. If I click it a second time, nothing more happens.
My Ditto's _Send To_ menu does not offer sending to the OS clipboard.

**Comment by PzaThief** [comment]
copy the item after click. click is just select step, so now you need to copy it.
and Ditto's Send To is about network sharing function not about OS clipboard.

**Comment by Chealer** [comment]
The clips are already copied (that is how they were added in Ditto). What I am asking is a way to send an "old clip" to the operating system's clipboard.

**Comment by PzaThief** [comment]
Yes. it's what you want. if you copy the clip again, Ditto will send it to OS and do not copy again.

**Comment by Chealer** [comment]
Sorry, I cannot make sense of your sentence.
Could you please write the method you propose in a language other than English? I understand French, English, Spanish and Ido.

**Comment by ariefcfa** [comment]
@Chealer maybe like this: (by the way i change Ditto shortcut `` ctrl + ` ``  to `ctrl + alt +v` 


https://github.com/sabrogden/Ditto/assets/34770799/3b956e50-559f-48c0-a3b1-73d4ccc0ed67

**Comment by Chealer** [comment]
Ooooohh😮, that's a lot clearer @ariefcfa , thank you! That does work for me.
So the "Copy Selection" command sends the clip to the OS clipboard and does nothing else? If so, is it normal that it cannot be used with the mouse (only with a shortcut)?

**Comment by ariefcfa** [comment]
@Chealer no worries :). 

> So the "Copy Selection" command sends the clip to the OS clipboard and does nothing else?

i think so

> ... cannot be used with the mouse (only with a shortcut)?

i tested it, and it can be used with the mouse. for example:

https://github.com/sabrogden/Ditto/assets/34770799/20e3115a-a9ed-40a7-a64e-90dd27cb7c6f

**Comment by Chealer** [comment]
Oooh😮, I see, thanks @ariefcfa 
But what I meant is that there is no way to send to the Windows clipboard via the clip's contextual menu (right-click), right? There is no way to do that by default?
